doc/sphinx/source/community/release_strategy.rst

```diff
@@ -192,21 +192,16 @@ These are the detailed steps to take to make a release.
    - Any contributor is welcome to add issues or pull requests that they intend to work on themselves to a milestone.
 
 
-#. ESMValCore feature freeze
+#. ESMValCore feature freeze, testing and release
 
    - A release branch is created and branch protection rules are set up so only the release manager (i.e. the person in charge of the release branch) can push commits to that branch.
-   - The creation of the release branch is announced to the ESMValTool development team along with the procedures to use the branch for testing and making last-minute changes (see next step)
-
-
-#. Some additional testing of ESMValCore
-
-   - Run all the recipes (optionally with a reduced amount of data) to check that they still work
+   - Make a release candidate with the release branch. Note that the :ref:`ESMValCore release instructions <esmvalcore:how-to-make-a-release>` are currently outdated and pending to be modified,
+     but they are analogous to the ESMValTool instructions listed in :ref: `How to make a release`. 
+   - Run all the recipes (optionally with a reduced amount of data) to check that they still work with the release candidate
```

Comment:
This is explained in much more detail in step 4. Could you have a look if it still makes sense to have step 4 where it is now? Do we need new/different steps?